src/server/storage/public-projects.js

```diff
@@ -27,7 +30,8 @@
                 primaryRoleName: project.activeRole,
                 roleNames: Object.keys(project.roles),
                 thumbnail: activeRole.Thumbnail[0],
-                notes: activeRole.Notes[0]
+                notes: activeRole.Notes[0],
+                services: _.uniq(extractRpcs(activeRole.SourceCode))
```

Comment:
do you mean to only get the rpcs from the activeRole here? I would have expected it to behave the same as the examples (retrieve the rpcs used across all roles)